packages/select/test/selectComponentSuite.tsx

```diff
@@ -36,7 +36,7 @@ export function selectComponentSuite<P extends IListItemsProps<IFilm>, S>(
 
     describe("common behavior", () => {
         it("itemRenderer is called for each child", () => {
-            const wrapper = render({ ...testProps, resetOnQuery: false });
```

Comment:
@switz I was able to remove this prop! default props do the right thing now. the one below stays, which makes sense cuz those tests are about specific reset logic.